source/_integrations/signal_messenger.markdown

```diff
@@ -65,6 +65,34 @@ recipients:
   type: string
 {% endconfiguration %}
 
+
+## Sending messages to Signal to trigger events
+In this example, we will be using signal-cli-rest-api to as a Home Assistant trigger. For example, you would be able to write 'light' to your Signal account linked to signal-cli-rest-api and have Home Assistant adjust the lights for you. To accomplish this, you will need to edit the `rest.yaml` configuration of Home Assistant, add the following resource:
+```- resource: "http://127.0.0.1:8080/v1/receive/<number>"
```

Comment:
Please start it on a new line.

Reply:
Thank you for your feedback. I'll review them!